doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+.. _guide_label:
+
+Guide to Simpleline
+===================
+
+Simpleline is the text user interface framework completely written in Python 3 with a possibility to
+have :ref:`non-python event loop <event_loops_label>`. With exception of optional event loops
+Simpleline doesn't have almost any dependency to external libraries.
+
+This UI is simple and easy to use. It is designed to be used with line-based machines and tools
+(e.g. serial console) so that every new line is appended to the bottom of the screen.
+Printed lines are never rewritten!
+
+Basic components
+----------------
+
+For every application the following parts are always required and a developer of an application
+will work with them.
+
+* The :class:`App <simpleline.App>` static class to initialize and run application.
+* The :class:`ScreenHandler <simpleline.render.screen_handler.ScreenHandler>` static class for
+  scheduling screens.
+* The :class:`UIScreen <simpleline.render.screen.UIScreen>` based classes to create screens which
+  will form the application.
+* :ref:`Widgets <widgets_label>` to show anything on the screens.
+* :ref:`Containers <containers_label>` to position widgets on the screen.
+
+It may look difficult to work with so many components but it is simple. Look on the next section to
```

Comment:
`It may look difficult to work with so many components but it is simple.` The main reason I take issue with this statement is because
(a) you created this, so of course it is simple to you -- you are biased. :)
(b) you are stating an opinion as a fact.
I'm not sure how I'd rewrite this, but maybe think about another way to convey your message, that this *should* be a simple library to work with.

`Look on the next` -> `Look at the next`

Reply:
I removed that part. I was thinking that this shouldn't be in docs at all. It's not a fact.